services/src/test-library/src/main/java/org/openkilda/testing/Constants.java

```diff
@@ -25,7 +25,7 @@
     public static final Integer RULES_DELETION_TIME = 5;
     public static final Integer RULES_INSTALLATION_TIME = 5;
     public static final Integer HEARTBEAT_INTERVAL = 10;
-    public static final Integer MAX_DEFAULT_METER_ID = 15;
+    public static final Integer MAX_DEFAULT_METER_ID = 31;
```

Comment:
Maybe it's better to use `MAX_SYSTEM_RULE_METER_ID` from MeterId class?

Reply:
Will be fixed in #1973. Thanks.